benchmark/timers/timers-timeout-pooled.js

```diff
@@ -2,22 +2,26 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  thousands: [500],
+  millions: [10],
 });
 
 function main(conf) {
-  const iterations = +conf.thousands * 1e3;
-  var count = 0;
+  const iterations = +conf.millions * 1e6;
+
+  // Function tracking on the hidden class in V8 can cause misleading
+  // results in this benchmark if only a single function is used —
+  // alternate between two functions for a fairer benchmark
+
+  function cb() {}
+  function cb2() {}
+
+  process.on('exit', function() {
+    bench.end(iterations / 1e6);
```

Comment:
Not loving this... It's async, so it could add noise, also having the callbacks have side effects improves the chances they are not optimized away.
I'd rather have duplicate `cb()` code, or have `cb` just `count++` and `cb2` test for exit.

Reply:
Addressed with duplicate code.